package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "bedrock-vc-status-list-context",
+  "version": "1.0.0",
```

I think this has to be less than 1.0 to allow for a major version publish at 1.0 -- but could be wrong about that.

```suggestion
  "version": "0.0.1",
```

---

Fixed a92f012